Oh, there is another error:

```
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [23 lines of output]
      CC env var: 'gcc'
      CFLAGS env var: <missing>
      extra_compile_args: ['-Werror', '-DMODULE_NAME=_test_limited_c11_cext', '-std=c11', '-DPy_LIMITED_API=0x30d00a5']
      running bdist_wheel
      running build
      running build_ext
      building '_test_limited_c11_cext' extension
      creating build
      creating build/temp.macosx-14.0-arm64-cpython-313-pydebug
      gcc -fno-strict-overflow -Wsign-compare -g -Og -Wall -I/Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/build/test_python_59797æ/tempcwd/env/include -I/Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/Include -I/Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build -c extension.c -o build/temp.macosx-14.0-arm64-cpython-313-pydebug/extension.o -Werror -DMODULE_NAME=_test_limited_c11_cext -std=c11 -DPy_LIMITED_API=0x30d00a5
      In file included from extension.c:7:
      In file included from /Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/Include/Python.h:58:
      /Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/Include/object.h:317:22: error: call to undeclared function '_Py_atomic_load_uint32_relaxed'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
          uint32_t local = _Py_atomic_load_uint32_relaxed(&ob->ob_ref_local);
                           ^
      /Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/Include/object.h:321:25: error: call to undeclared function '_Py_atomic_load_ssize_relaxed'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
          Py_ssize_t shared = _Py_atomic_load_ssize_relaxed(&ob->ob_ref_shared);
                              ^
      /Users/ec2-user/buildbot/buildarea/pull_request.itamaro-macos-arm64-aws.macos-with-brew.nogil/build/Include/object.h:432:5: error: call to undeclared function '_Py_atomic_store_ssize_relaxed'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
          _Py_atomic_store_ssize_relaxed(&ob->ob_size, size);
          ^
      3 errors generated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
```

--- a/Lib/test/test_cext/__init__.py
+++ b/Lib/test/test_cext/__init__.py
@@ -40,9 +40,11 @@ class TestExt(unittest.TestCase):
     def test_build_c99(self):
         self.check_build('_test_c99_cext', std='c99')
 
+    @unittest.skipIf(support.Py_GIL_DISABLED, 'incompatible with Free Threading')
     def test_build_limited(self):
         self.check_build('_test_limited_cext', limited=True)
 
+    @unittest.skipIf(support.Py_GIL_DISABLED, 'broken for now with Free Threading')
     def test_build_limited_c11(self):
         self.check_build('_test_limited_c11_cext', limited=True, std='c11')
 

--- a/Lib/test/test_cext/setup.py
+++ b/Lib/test/test_cext/setup.py
@@ -11,7 +11,7 @@ from setuptools import setup, Extension
 
 
 SOURCE = 'extension.c'
-if not support.MS_WINDOWS and not sysconfig.get_config_var('Py_GIL_DISABLED'):
+if not support.MS_WINDOWS and not support.Py_GIL_DISABLED:
     # C compiler flags for GCC and clang
     CFLAGS = [
         # The purpose of test_cext extension is to check that building a C

